master/getting-started/bare-metal/installation/binary-mgr.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Binary install with package manager
+canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/bare-metal/installation/binary-mgr'
+---
+
+
+## Prerequisite
+
+1. [Install calicoctl as a binary](/{{page.version}}/usage/calicoctl/install#installing-calicoctl-as-a-binary).
+
+1. [Configure calicoctl to connect to etcd](/{{page.version}}/usage/calicoctl/configure/).
+
+
+## Installing Felix
+
+{% include ppa_repo_name %}
+
+There are several ways to install Felix.
+
+-   If you are running Ubuntu 14.04 or 16.04, you can install from our PPA:
+
+        sudo add-apt-repository ppa:project-calico/{{ ppa_repo_name }}
+        sudo apt-get update
+        sudo apt-get upgrade
+        sudo apt-get install calico-felix
+
+-   If you are running a RedHat 7-derived distribution, you can install
+    from our RPM repository:
+
+        cat > /etc/yum.repos.d/calico.repo <<EOF
+        [calico]
+        name=Calico Repository
+        baseurl=http://binaries.projectcalico.org/rpm/{{ ppa_repo_name }}/
+        enabled=1
+        skip_if_unavailable=0
+        gpgcheck=1
+        gpgkey=http://binaries.projectcalico.org/rpm/{{ ppa_repo_name }}/key
+        priority=97
+        EOF
+
+        yum install calico-felix
+
+
+Until you initialize the database, Felix will make a regular log that it
+is in state "wait-for-ready". The default location for the log file is
+`/var/log/calico/felix.log`.
+
+## Initializing the etcd database
+
+You should configure a `node` resource for each
+host running Felix.  In this case, the database is initialized after
+creating the first `node` resource.  For a deployment that does not include
+the {{site.prodname}}/BGP integration, the specification of a node resource just requires
+the name of the node;  for most deployments this will be the same as the
+hostname.
+
+```
+cat << EOF | calicoctl create -f -
+- apiVersion: projectcalico.org/v3
+  kind: Node
+  metadata:
+    name: <node name or hostname>
+EOF
+```
+
+If you check the Felix logfile after this step, the logs should
+transition from periodic notifications that Felix is in state
+"wait-for-ready" to a stream of initialization messages.
```

Comment:
Since the section is about initializing the etcd datastore, I think at the end of it, they should have accomplished this. It seems that if Felix is still in `wait for ready` then that wouldn't be the case. Perhaps the procedure has a prerequisite that they have an orchestrator sharing their etcd and that the orchestrator has initialized the datastore?

Reply:
Well, I am not sure, but I think that what this doc says, on its own, is not sufficient.  Do you know if anyone is scheduled to test this install before 3.1?